src/Controller/OgAutocompleteController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Drupal\og\Controller;
+
+use Drupal\Component\Utility\Crypt;
+use Drupal\Component\Utility\Tags;
+use Drupal\Component\Utility\Unicode;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Entity\EntityAutocompleteMatcher;
+use Drupal\Core\KeyValueStore\KeyValueStoreInterface;
+use Drupal\Core\Site\Settings;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+/**
+ * Defines a route controller for entity autocomplete form elements.
```

Comment:
@jludwig 
If we keep this class in the end, we should differ the class comment compared to `core/modules/system/src/Controller/EntityAutocompleteController.php` with some OG-specifics.

Reply:
This became
 * Defines a route controller for OG autocomplete form elements.